src/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/DiagnosticListener.cs

```diff
@@ -353,4 +359,45 @@ public void Dispose()
 
         #endregion
     }
+
+    /// <summary>
+    /// Current this EventSource is only here so that the Debugger can inject code using Function evaluation
+    /// We may add actual logging requests as well at some point.   This can be removed if the debugger  no 
+    /// longer needs it.   
+    /// </summary>
+    [EventSource(Name = "Microsoft-Diagnostics-DiagnosticSource")]
+    class DiagnosticSourceEventSource : EventSource
+    {
+        static public DiagnosticSourceEventSource Logger = new DiagnosticSourceEventSource();
```

Comment:
Nit: elsewhere this file (and in general in the repo) orders visibility before static, so ```public static```